### PR TITLE
Feature/add tag to workflow

### PR DESCRIPTION
**Pull request by gpoleze**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/prod-efax-service-build-deploy.yaml [hunk 1]
```diff
@@ -3,13 +3,19 @@ name: PROD Deploy of Efax Webservice Image
 on:
   workflow_dispatch:
     branches: [main]
```
Comment (KDNeufeld) [inline .github/workflows/prod-efax-service-build-deploy.yaml:5]: Now that there is a tag input, is this line still needed?

File: .github/workflows/test-efax-service-build-deploy.yaml [hunk 2]
```diff
@@ -3,13 +3,19 @@ name: TEST Deploy of Efax Webservice Image
 on:
   workflow_dispatch:
     branches: [main]
```
Comment (KDNeufeld) [inline .github/workflows/test-efax-service-build-deploy.yaml:5]: Is this line still needed or desired?  I don't think we ever want to (or should) build from main, only a specific tag version.

**Comment by KDNeufeld** [comment]
This is looking a lot better! Great work.  I like the idea of building from a specific tag for TEST or PROD.

File: .github/workflows/test-efax-service-build-deploy.yaml (hunk 2 above)
Comment (gpoleze) [reply .github/workflows/test-efax-service-build-deploy.yaml]: We do not, and I removed it.

File: .github/workflows/prod-efax-service-build-deploy.yaml (hunk 1 above)
Comment (gpoleze) [reply .github/workflows/prod-efax-service-build-deploy.yaml]: We do not, and I removed it.

**Review by KDNeufeld — APPROVED** [review]
LGTM

**Pull request merged**